.azure/azure-pipelines.wanperf.yml

```diff
@@ -92,3 +96,5 @@ stages:
       rttMs: ${{ parameters.rttMs }}
       pacing: ${{ parameters.pacing }}
       logProfile: ${{ parameters.logProfile }}
+      ${{ if ne(parameters.extraArgs, 'none') }}:
```

Comment:
Any specific reason for defaulting to `none` and checking it its not that? All the other extra args we use just leave it empty and pass the empty value thorough.

Reply:
AZP doesn't allow empty input fields. You're required to put a value in it, so I use `none` as that value by default.